google/cloud/spanner_v1/instance.py

```diff
@@ -94,10 +95,12 @@ class Instance(object):
 
     :type node_count: int
     :param node_count: (Optional) Number of nodes allocated to the instance.
+                        Cannot be set if processing_units is set.
```

Comment:
nit: use `processing_units` for the docs render. Also applies to processing_units docstring below